### ISSUE TITLE
APIv7 --> APIv8

### ISSUE THREAD

**Issue by c0lumb0** [marklagendijk/node-toogoodtogo-watcher#210]
Seems like TGTG updated their API once more since I got this error recently

`Error during request:`
`POST https://apptoogoodtogo.com/api/item/v7/`
`{`
    `"favorites_only": true,`
    `"origin": {`
        `"latitude": 52.5170365,`
        `"longitude": 13.3888599`
    `},`
    `"radius": 200,`
    `"user_id": "123456789"`
`}`

`HTTPError: Response code 403 (Forbidden)`
    `at Request.<anonymous> (C:\toogoodtogo-watcher\node_modules\got\dist\source\as-promise\index.js:118:42)`
    `at processTicksAndRejections (node:internal/process/task_queues:96:5)`

Changing this

`return api.post("item/v7/"`

to

`return api.post("item/v8/"`

in api.js fixed the issue.

**Comment by basti1499** [comment]
I am experiencing another HTTP 403 error, but this one occurs during the token refresh:

```
Error during request:
POST https://apptoogoodtogo.com/api/auth/v3/token/refresh
{
    "refresh_token": "e30.eyJzdWIiOiI3NDM3ODQ3MSIsImV4cCI6MTcwNzgyOTAyMSwidCI6Ill2VVFETEcwU1lxWndIWEFqeWlYTFE6MDowIn0.27bFjP3dO0CI3PXttLhF-QMIdz6LlQX-yh64xFux_Rc"
}

HTTPError: Response code 403 (Forbidden)
    at Request.<anonymous> (/usr/lib/node_modules/toogoodtogo-watcher/node_modules/got/dist/source/as-promise/index.js:118:42)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

Haven't been able to figure out yet how to fix it.
Anyone else with a similar problem or even a fix?

**Comment by marklagendijk** [comment]
Released in `3.9.2`
